projects/demo/src/modules/utils/tokens/examples/7/index.html

```diff
@@ -1 +1,4 @@
-<p i18n>Provide an array of 12 strings for localized names for months to be used in TuiMonthPipe</p>
+<p i18n>
+    A token with a factory. It compares the viewport width and the value from the TUI_MEDIA token to detect users with
+    mobile resolution
+</p>
```

Comment:
Let's add an `async` pipe with that stream so people could see it in action.

Reply:
this token also lives in v3. so should we raise another issue for this change?

Reply:
No need for a separate issue, but if you can make a quick PR, that would be great :)

Reply:
done. if it works for you let me know. I'll create the same PR for the current version